src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.HighPriorityProcessor.cs

```diff
@@ -49,6 +49,10 @@ private sealed class HighPriorityProcessor : IdleProcessor
                         Start();
                     }
 
+                    protected override void OnPaused()
+                    {
+                    }
```

Comment:
the base method is abstract so that i had to go to all types to make it clear in them that they're doing nothing (like before).